src/cleanvision/imagelab.py

```diff
@@ -211,6 +314,42 @@ def report(
         examples_per_issue: Optional[int] = None,
         verbosity: int = 1,
     ) -> None:
+        """Prints a summary of issues found in the dataset with their example images from the dataset.
+        By default, if no arguments are specified, it reports the top issues found in the dataset.
+
+        Parameters
+        ----------
+        issue_types : List[str], optional
+            List of issue types to report
+
+        num_top_issues : int, default=3
```

Comment:
Should the default value be much larger, like 10?
I think somebody can easily understand 5-10 types of issues. Isn't it important to make somebody aware of 5 issues if they are all pretty severe in their dataset?

@elisno

Reply:
I don't think it is worth switching to class yet, we don't want to delay anymore on this project. I've added github issue [here](https://github.com/cleanlab/cleanvision/issues/102) where @elisno can provide more pointers and the discussion can continue.

Reply:
> Should we have a (+123 other types of issues) line at the end of the report if we still worry about users "missing" severe issue types?

Not in my opinion. The things that are over max_prevalence are not issues. If the dataset is all black/white, or all blurry, these are basically just not issues in the dataset that need to be reported to user. User can still easily figure these out, by looking at Imagelab attributes (or just browsing their data), but these are very likely low-priority for the user. 

In Imagelab & Datalab reports, we **always** want to err on the side of showing **less** not more. The purpose of these is to focus attention where it is needed most.

Reply:
Changed the `imagelab.report()`
- Removed num_top_issues as it was a useless argument, number of issues output can be increased by verbosity.
- For `max_prevalence`, first filter out issues in both `issue_types` argument or if no issue types are provided according to max_prevalence and then report the issues.
- Added a print statement to indicate if there were more issues found in the dataset and how to view them.